docker/Dockerfile.nvidia.clang

```diff
@@ -0,0 +1,48 @@
+
+##############################################################
+# This Dockerfile contains NVIDIA HPC SDK (nvc, cuda, OpenMPI) and LLVM-15 for openmp offloading for Devito
+##############################################################
+FROM devito:nvidia
+
+## Environment Variables for OpenMP Builds
+ENV DEVITO_ARCH="clang"
+ENV DEVITO_LANGUAGE="openmp"
+ENV DEVITO_PLATFORM="nvidiaX"
+ENV DEVITO_LOGGING="DEBUG"
+ENV OMPI_CC="clang"
+ENV CFLAGS=
+
+## INSTALL CLANG PREREQUISITES
+RUN apt-get -y update && \
+    apt-get install -y -q libelf-dev libffi-dev cmake git
+
+## INSTALL CLANG 13
+WORKDIR /
+RUN git clone --depth 1 https://github.com/llvm/llvm-project.git
```

Comment:
so this is latest master, we should pin a specific version no? this is otherwise very likely to break? or at least it did break regularly, in the past